src/OpenTelemetry.AutoInstrumentation/Properties/AssemblyInfo.NetFramework.cs

```diff
@@ -18,7 +18,7 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyKeyFile("keypair.snk")]
+[assembly: AssemblyKeyFile("../keypair.snk")]
```

Comment:
> @dszmigielski Files are moved into `src/Shared/OpenTelemetry.AutoInstrumentation.Logging/` without a project file. It's best practice to create shared project and place files in it.
> 
> It is not a blocker and we could create an issue to track it.

I would leave it as is for the moment. It will require build changes, as we would probably not want that to be built, and for sure we would not want the output dll to be included into tracer.